zubhub_frontend/zubhub/src/assets/js/styles/views/projects/projectsStyles.js

```diff
@@ -108,20 +137,44 @@ const styles = theme => ({
   },
   heroMessagePrimaryStyle: {
     fontSize: '2.2rem',
-    fontWeight: 1000,
+    fontWeight: '1000',
   },
   MessagePrimaryStyle: {
     fontSize: '2.5rem',
     fontWeight: 1000,
   },
-
+  heroMainMessageSecondaryStyle: {
+    fontSize: '4.5rem',
+    fontWeight: '700',
+    [theme.breakpoints.down('sm')]: {
+      fontSize: '2.5rem'
+    },
+  },
+  heroMainMessagePrimaryStyle: {
+    fontSize: '2.2rem',
+    fontWeight: 500,
+    [theme.breakpoints.down('sm')]: {
+      fontSize: '1.2rem'
+    },
+  },
   heroButtonStyle: {
     textTransform: 'none',
-    padding: '0 1em',
-    fontSize: '1.2rem',
+    fontSize: '1.456rem',
```

Comment:
Why do we need 3 decimal accurate values? How have these been calculated? I feel if we need such fine adjustments, then there must be some underlying issue

Reply:
No, I don't think there is any underlying issue. My reasoning is the same as above ☝️